nexuse2e-webapp/src/main/frontend/src/assets/i18n/en.json

```diff
@@ -22,7 +22,7 @@
     "conversations": "Conversations",
     "messageId": "Message ID",
     "conversationId": "Conversation ID",
-    "participantId": "Participant ID",
+    "participantId": "Partner ID",
```

Comment:
Why translate participantId to Partner ID? This sound like to different things and having this not named uniformly, can lead to confusions. Is it useful to rename the participantId label to partnerid?